rhel8/profiles/cis.profile

```diff
@@ -813,69 +814,75 @@ selections:
 
 
     ### 5.3.1 Create custom authselectet profile (Scored)
-
+    # NEEDS RULE - https://github.com/ComplianceAsCode/content/issues/5530
 
     ### 5.3.2 Select authselect profile (Scored)
-
+    # NEEDS RULE - https://github.com/ComplianceAsCode/content/issues/5531
 
     ### 5.3.3 Ensure authselect includes with-faillock (Scored)
-
+    # NEEDS RULE - https://github.com/ComplianceAsCode/content/issues/5532
 
     ## 5.4 Configure PAM
 
     ### 5.4.1 Ensure password creation requirements are configured (Scored)
-
+    # NEEDS RULE - https://github.com/ComplianceAsCode/content/issues/5533
 
     ### 5.4.2 Ensure lockout for failed password attempts is
     ###       configured (Scored)
-
+    # NEEDS RULE - https://github.com/ComplianceAsCode/content/issues/5534
```

Comment:
I think we could cover this with rules from the locking_out_password_attempts group

Reply:
Likely! Feel free to self-assign that ticket and review/patch as applicable.